nginx-app/recipes/rsyslogs.rb

```diff
@@ -0,0 +1,7 @@
+include_recipe 'rsyslogd'
+
+template '/etc/rsyslog.d/45-nginx.conf' do
```

Comment:
moved this from 49-nginx.conf to 45-nginx.conf to come before loggly 49-loggly.conf